### ISSUE TITLE
Deeper Lua integration

### ISSUE THREAD

**Issue by Menta2L** [pavel-odintsov/fastnetmon#320]
Its not a issue but like future request.
The idea to use Lua for packet parsing or alert send is very nice . Probably is not a bad idea to be extended.
In addition to that i think that kind of "modular system" may be useful too specially when the project comes to the Implement application level flood monitoring (http, dns, https).

I actually wrote some code in that direction in my fork and if you are interested i can make a pull request

**Comment by pavel-odintsov** [comment]
Hello!

Nice idea. Could you describe what task you want to solve with LUA?

**Comment by Menta2L** [comment]
Hello,
here is few examples.
1. Can be used as configuration file no sense to use a key value config when you have lua support anyway.
2. Exporting a logger to lua so you can debug your scripts in realtime.
3. Can be used as hook as packet  post processing ( for example for service based statistics)
4. Hook for notifications and bans so you may have "build-in"  functions for that.
5. Insert  or query the patricia table from lua script 
6. Module based initialization For example in netflow lua hook you can load /usr/src/fastnetmon/src/tests/netflow_exclude.json just once during module initialization.

**Comment by pavel-odintsov** [comment]
Hello, very good! But for this approach I need deeper LUA integration. But we have some troubles because LUAJIT offered in distributions is broken or absent sometimes: https://github.com/FastVPSEestiOu/fastnetmon/blob/master/docs/LUA_SUPPORT.md. 

I thinking about LUAJIT integration as Git Sub module but not sure about any ETA.

**Comment by pavel-odintsov** [comment]
Hello :)

I have finished deep integration of LUA and it's a mandatory part of FastNetMon now :)  

There are copy of maillist message:

Hello, Community!

We have integrated FastNetMon with awesome and lightning fast lua jit
interpreter.

We already use it for sFLOW and NetFlow packet filtering and looking
forward for attack detection engine in LUA.

So, we also spent some time for fastnetmon_install.pl script
enhancements and it become more readable :)

Testers are welcome!

Modern way to get all tasty features with single script:
wget https://raw.githubusercontent.com/FastVPSEestiOu/fastnetmon/master/src/fastnetmon_install.pl
-Ofastnetmon_install.pl
sudo perl fastnetmon_install.pl --use-git-master
